alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/solr/sharding/SolrShardingMetricsContainer.java

```diff
@@ -17,7 +18,7 @@
 public class SolrShardingMetricsContainer {
 
     private ShardRegistry shardRegistry;
-    private HashMap<Floc, HashMap<Shard, HashSet<ShardState>>> rawData;
+    private Map<Floc, HashMap<Shard, HashSet<ShardState>>> rawData;
```

Comment:
FYI: required because the return type of `ShardRegistry#getFlocs()` changed from `HashMap<>` to `Map<>`:
https://github.com/Alfresco/alfresco-community-repo/blob/1eeac685614e8dce4f7d94535e73d6f6dd9a6930/repository/src/main/java/org/alfresco/repo/index/shard/ShardRegistry.java#L56
(And we should program towards the interface anyway.)

Reply:
Oh yes, this was possible even before Alfresco changed the return type.